src/main/core/manager.rs

```diff
@@ -732,7 +732,7 @@ impl<'a> Manager<'a> {
 
             // both ':' and ' ' are valid LD_PRELOAD separators
             for path in value.split(&[':', ' '][..]) {
-                let path = utility::tilde_expansion(path);
+                let path = utility::resolve_path(Path::new(path)).unwrap();
```

Comment:
I *don't* think we want `$PATH` interpolation for the `LD_PRELOAD` environment variable. We should keep the `tilde_expansion` helper and still use it here.